src/Microsoft.DocAsCode.AzureMarkdownRewriters/AzureBlockRules/AzureMigrationHtmlMetadataBlockRule.cs

```diff
@@ -82,7 +82,13 @@ private AzureHtmlMetadata GetAttributesFromHtmlContent(string htmlContent)
                 {
                     if (string.Equals(attribute.Name, "pageTitle", StringComparison.OrdinalIgnoreCase))
```

Comment:
"pageTitle" has already been converted to "title".

---
In reply to: [85067936](https://github.com/dotnet/docfx/pull/826#discussion_r85067936) [](ancestors = 85067936)